change/@griffel-core-40e0578a-53de-40b8-9ed2-958d4fe70437.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "minor",
```
Shouldn't this be a minor? as it's a new feature....